config/install/openam_api.settings.yml

```diff
@@ -4,3 +4,27 @@ openam_api_url: ""
 openam_api_username: ""
 openam_api_password: ""
 openam_api_timeout: 30
+openam_api_operations:
```

Comment:
this is defined as 'openam_api_operations.settings' in the schema! Are you seeing any issues with the mismatched naming convention?

Reply:
Updated.